docs/index.md

```diff
@@ -0,0 +1,6 @@
+# MoveIt Middleware Benchmark
+
+This is a package created for measuring middleware effects on various scenarios. This package contains some scenarios such as perception pipeline.
```

Comment:
I like that! Move it directly into the README.md file. The tutorials use and index.md file because it is used to build the website but for this repos, I think it is not necessary

Reply:
Done it in https://github.com/CihatAltiparmak/moveit_middleware_benchmark/pull/5/commits/57a5a2ce4de5ae5663a36a34fb412d9509c7e6d6 commit. I hope this one improves README.md. Btw i also added CI badge :smiley: